juju/docs
# Update documentation-builder to 1.6.0

## Done

- Update documentation-builder to 1.6.0
- Update metadata.yaml with favicon, logo url, and navigation styles

## QA

- Pull code
- Either change branch name to `devel`, or add whatever the name of the branch is to versions.txt
- Check the new styling has been applied
